config.py

```diff
@@ -92,6 +92,80 @@ def load(self):
 The head of the HAProxy config. This contains global settings
 and defaults.
 '''))
+        self.add_template(
```

Comment:
Duplicating all of this violates the DRY principle, and it seems unnecessary. How about we just have the Lua endpoint enabled all the time? I think it's okay if it returns an empty response when the map isn't enabled.